torch/fx/experimental/symbolic_shapes.py

```diff
@@ -2902,24 +2927,6 @@ def issue_guard(guard: ShapeGuard) -> None:
                 r = self.runtime_var_to_range.get(symbol)
                 if r is None:
                     r = self.var_to_range[symbol]
-
```

Comment:
I checked that this is effectively dead code: https://github.com/pytorch/pytorch/pull/107628
We always keep the ranges consistent.